### PR TITLE
Add route from centroid to source geom

### PR DESCRIPTION
**Pull request by GideonTargomo**

### CODE REVIEW

**Comment by GideonTargomo** [comment]
Add `routeFromCentroid` option for routing geometries

File: src/main/java/com/targomo/client/api/geo/AbstractGeometry.java [hunk 1]
```diff
@@ -51,6 +60,13 @@ public void setCrs(Integer crs) {
         this.crs = crs;
     }
 
+    /**
+     * @param routeFromCentroid if true route from/to the centroid of this geometry when no intersections found
+     */
+    public void setRouteFromCentroid(Boolean routeFromCentroid) {
```
Comment (HLucasTargomo) [inline src/main/java/com/targomo/client/api/geo/AbstractGeometry.java:66]: I think we should use the primitive `boolean` here

File: src/main/java/com/targomo/client/api/geo/AbstractGeometry.java [hunk 2]
```diff
@@ -14,13 +14,15 @@
 
     private Integer crs;
     private String data;
+    private Boolean routeFromCentroid = true;
```
Comment (HLucasTargomo) [inline src/main/java/com/targomo/client/api/geo/AbstractGeometry.java:17]: I think we should use the primitive `boolean` here

File: src/main/java/com/targomo/client/api/geo/AbstractGeometry.java [hunk 3]
```diff
@@ -14,13 +14,15 @@
...
 
     public AbstractGeometry() {} //For jackson test
 
-    public AbstractGeometry(String id, Integer crs, String data, LocationProperties locationProperties) {
+    public AbstractGeometry(String id, Integer crs, String data, Boolean routeFromCentroid, LocationProperties locationProperties) {
```
Comment (HLucasTargomo) [inline src/main/java/com/targomo/client/api/geo/AbstractGeometry.java:21]: same here

File: src/main/java/com/targomo/client/api/geo/AbstractGeometry.java (hunk 2 above)
Comment (GideonTargomo) [reply src/main/java/com/targomo/client/api/geo/AbstractGeometry.java]: switched

File: src/main/java/com/targomo/client/api/geo/AbstractGeometry.java (hunk 1 above)
Comment (GideonTargomo) [reply src/main/java/com/targomo/client/api/geo/AbstractGeometry.java:66]: switched

File: src/main/java/com/targomo/client/api/geo/AbstractGeometry.java (hunk 3 above)
Comment (GideonTargomo) [reply src/main/java/com/targomo/client/api/geo/AbstractGeometry.java]: switched

**Review by atomicbomberman — APPROVED** [review]

**Pull request merged**